README.md

```diff
@@ -65,13 +65,20 @@ With a simplified architecture and no use of CloudFormation, there are no limits
 
 ### Getting started
 
+Install the component:
+
+```
+npm i @sls-next/serverless-component -D
```

Comment:
You don't actually need to `npm install` the component. I've deliberately left this step out :)

Reply:
Thanks @danielcondemarin.
Can you tell me why I've got the error (see description screenshot) when running serverless deploy?

Reply:
That version is genuinely not in the registry:

![image](https://user-images.githubusercontent.com/1122442/89105815-66f92c00-d41c-11ea-8aed-73c95ad72bc3.png)

Reply:
oops :)
Have no idea where did I take that version from. Probably my Friday imagination :)

Is it true that `component: serverless-next.js` should be replaced with `component: "@sls-next/serverless-component@{version_here}"` across Readme file ?

Reply:
Yes that's true. serverless-next.js used to be the old name of the package before I created the org @sls-next :)